repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,378 @@
+{
+  "additionalProperties" : false,
+  "description" : "Marathon universe properties",
+  "properties" : {
+    "marathon" : {
+      "additionalProperties" : false,
+      "description" : "Marathon App Definition properties",
```

Comment:
I think that we should use a longer description.

@ssk2 can probably help with phrasing it.